packages/kbn-investigation-shared/src/schema/create_notes.ts

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
nit: file name to `create_note.ts`

Reply:
I'll do this in another PR so the CI does not start again 😅

Reply:
sure